scripts/circleci/test_entry_point.sh

```diff
@@ -10,21 +10,21 @@ if [ $((0 % CIRCLE_NODE_TOTAL)) -eq "$CIRCLE_NODE_INDEX" ]; then
   COMMANDS_TO_RUN+=('node ./scripts/prettier/index')
   COMMANDS_TO_RUN+=('node ./scripts/tasks/flow-ci')
   COMMANDS_TO_RUN+=('node ./scripts/tasks/eslint')
-  COMMANDS_TO_RUN+=('yarn test --maxWorkers=2')
+  COMMANDS_TO_RUN+=("yarn test --maxWorkers=2 --testResultsProcessor=${JEST_PROCESSOR}")
   COMMANDS_TO_RUN+=('./scripts/circleci/check_license.sh')
   COMMANDS_TO_RUN+=('./scripts/circleci/check_modules.sh')
   COMMANDS_TO_RUN+=('./scripts/circleci/test_print_warnings.sh')
   COMMANDS_TO_RUN+=('./scripts/circleci/track_stats.sh')
 fi
 
 if [ $((1 % CIRCLE_NODE_TOTAL)) -eq "$CIRCLE_NODE_INDEX" ]; then
-  COMMANDS_TO_RUN+=('yarn test-prod --maxWorkers=2')
+  COMMANDS_TO_RUN+=("yarn test-prod --maxWorkers=2 --testResultsProcessor=${JEST_PROCESSOR}")
 fi
 
 if [ $((2 % CIRCLE_NODE_TOTAL)) -eq "$CIRCLE_NODE_INDEX" ]; then
   COMMANDS_TO_RUN+=('./scripts/circleci/build.sh')
-  COMMANDS_TO_RUN+=('yarn test-build --maxWorkers=2')
-  COMMANDS_TO_RUN+=('yarn test-build-prod --maxWorkers=2')
+  COMMANDS_TO_RUN+=("yarn test-build --maxWorkers=2 --testResultsProcessor=${JEST_PROCESSOR}")
+  COMMANDS_TO_RUN+=("yarn test-build-prod --maxWorkers=2 --testResultsProcessor=${JEST_PROCESSOR}")
```

Comment:
Same question here. Do these need to generate unique reports?

Reply:
Indeed, no need for unique reports. I am removing this, and keeping the reporting only in line 13.